soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6240,6 +6240,8 @@ s32 func_8083E5A8(Player* this, GlobalContext* globalCtx) {
                         this->actor.colChkInfo.damage = 0;
                         func_80837C0C(globalCtx, this, 3, 0.0f, 0.0f, 0, 20);
                         Player_SetPendingFlag(this, globalCtx);
+                        Message_StartTextbox(globalCtx, 0xF8, NULL);
+                        Audio_PlayFanfare(NA_BGM_SMALL_ITEM_GET);
```

Comment:
does this take the longer fanfares for quest items stuff into account?

Reply:
This is a bit of code that only ever runs when you get an ice trap, so it won't have longer fanfares. I added the fanfare to bring it in line with getting an ice trap from a skulltula token for example.